docs/source/changelog.md

```diff
@@ -8,6 +8,21 @@ command line for details.
 
 ## 15.0
 
+### 15.0.1
+
+#### Bugs fixed
+
+- [Bitbucket] Fix for changes to bitbucket API - /teams removed and /workspaces to be used [#477](https://github.com/jupyterhub/oauthenticator/pull/477) ([@Marcalberga](https://github.com/Marcalberga))
+- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps follow-up [#517](https://github.com/jupyterhub/oauthenticator/pull/517) ([@GeorgianaElena](https://github.com/GeorgianaElena))
+- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps [#516](https://github.com/jupyterhub/oauthenticator/pull/516) ([@GeorgianaElena](https://github.com/GeorgianaElena))
```

Comment:
Pff, what a lack of inspiration I had. @consideRatio, do you think it makes more sense to have the follow up PR be last?
like in the suggestion below.

```suggestion
- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps [#516](https://github.com/jupyterhub/oauthenticator/pull/516) ([@GeorgianaElena](https://github.com/GeorgianaElena))
- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps follow-up [#517](https://github.com/jupyterhub/oauthenticator/pull/517) ([@GeorgianaElena](https://github.com/GeorgianaElena))
```

Reply:
Oh i think you should leave it as generated by github-activity and possibly update titles/labels to ingluence how its generated. To make further tweaks seems too much work.

If you do want, feel free to make tweaks though.